terraform/environments/core-sandbox/playground.tf

```diff
@@ -19,16 +19,16 @@ module "ram-principal-association" {
 
 }
 
-data "aws_ami_ids" "example" {
-  owners     = ["self"]
-  name_regex = "(?!oracle-linux-5.11)*"
-  # filter {
-  #   name   = "name"
-  #   values = ["oracle-linux-5.11*"]
-  # }
-
-}
-
-output "amis" {
-  value = data.aws_ami_ids.example.ids
-}
\ No newline at end of file
+# data "aws_ami_ids" "example" {
```

Comment:
Comments here should either be removed, or the regex should be fixed as this isn't relevant to the purpose of this PR

Reply:
That's the section I commented out to get the plan to run. We need to look at it to see if the code can be amended to work correctly.

Reply:
Removed from the PR